src/core/dev/hw_queue_tx.cpp

```diff
@@ -1290,11 +1260,7 @@ void hw_queue_tx::post_nop_fence(void)
     cseg->qpn_ds = htobe32((m_mlx5_qp.qpn << MLX5_WQE_CTRL_QPN_SHIFT) | 0x01);
     cseg->fm_ce_se = MLX5_FENCE_MODE_INITIATOR_SMALL;
 
-    store_current_wqe_prop(nullptr, SQ_CREDITS_NOP, nullptr);
-
-    ring_doorbell(1);
-
-    update_next_wqe_hot();
+    submit_wqe(nullptr, SQ_CREDITS_NOP, 1U, nullptr, false);
```

Comment:
what is this magic number 1? why do we need it and do we need a macro (define) for this?